libvncserver/tight.c

```diff
@@ -1516,7 +1542,7 @@ DEFINE_IDX_ENCODE_FUNCTION(32)
 #define DEFINE_MONO_ENCODE_FUNCTION(bpp)                                \
                                                                         \
 static void                                                             \
-EncodeMonoRect##bpp(uint8_t *buf, int w, int h) {                       \
+EncodeMonoRect##bpp(uint8_t *buf, int w, int h, uint##bpp##_t msk) {    \
```

Comment:
Maybe you can give msk a more speaking name as there already is a local var mask.

Reply:
Would rgb_mask satisfy?

Reply:
Yes!